lib/offset.rb

```diff
@@ -0,0 +1,23 @@
+class Offset
+  attr_reader :a_offset, :b_offset, :c_offset, :d_offset
+  def initialize
```

Comment:
might need to come back and add @date instance variable so that it can be customized later incase anything other than current date will be used.